src/main/cpp/t07_swap.cpp

```diff
@@ -19,5 +19,23 @@
 using namespace std;
 
 int t07_swap() {
+	int n;
+	cin >> n;
+	int mas[1000];
+	for (int i = 0; i < n; i++) {
+		cin >> mas[i];
+	}
+	int numb = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'numb' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4532058)